### ISSUE TITLE
Apps have no acces to POD after instal via Nextcloud app store

### ISSUE THREAD

**Issue by kurijn** [pdsinterop/solid-nextcloud#122]
I installed the Solid app in Nextcloud (running on Debian LEMP stack, using yunohost). Using the [WebID URI](https://bebuysd.port0.org/nextcloud/apps/solid/@kurijn/profile/card#me) I find in the "Profile" section, I tried the listed "Solid Apps" but with no luck. In most cases there is no reaction at all when entering the WebID, and with the "Solid focus" app I get the error "Login with Solid failed, make sure that you introduced the correct Solid POD url and the POD server is running".
I'm unsure if I still need to follow any of the extra install instructions (https://github.com/pdsinterop/solid-nextcloud/blob/main/INSTALL.md) - I'm not building from source and the 2nd "troubleshooting" point seems to indicate Apache is required..? (I've an Nginx setup).

**Comment by Potherca** [comment]
Hi! My apologies for my slow reply. I've been on sick leave the last two weeks. (Sadly I caught the latest COVID variation).

Part of the issue with the applications not working stems from the fact that the Solid spec is moving forward and not all applications move with it. As we try to remain spec compliant, support of apps seems to have slipped.

We are aware of the issue bu do not (yet) have a solution for this, as manually testing all apps after every release is not feasible for us.

I have created https://github.com/orgs/pdsinterop/discussions/3 to facilitate the discussion in the hope of coming up with a viable solution to this problem

**Comment by ylebre** [comment]
The latest release (0.8.1) fixes a lot of the issues with compatibility with the solid apps. If you are still interested, it would merit another look ;)

**Comment by kurijn** [comment]
Thanks! I'll try it out!

**Comment by kurijn** [comment]
Ah, I think I've bad luck. I'm using nextcloud on a yunohost.org system, which is using a single-sign-on, so probably this causes the logging-in to be different. I just tried with liqid.chat but got 
"invalid json response body at [https://oneto.be/yunohost/sso/?r~..."
 I also tried https://noeldemartin.github.io/solid-focus/, https://notepod.vincenttunru.com/ with no reaction (haven't checked on http responses)
I tried https://empty-solid-app.5apps.com/ which gave me status: 200 though. ..?
